jobs/Backend/Task/ExchangeRateProvider/CurrencyCode.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+
+namespace ExchangeRateProvider
+{
+    public class CurrencyCode
+    {
+        public CurrencyCode(string code)
+        {
+            Code = code;
+        }
+
+        /// <summary>
+        /// Three-letter ISO 4217 code of the currency.
+        /// </summary>
+        public string Code { get; }
+
+        public override string ToString()
+        {
+            return Code;
+        }
+
+        public override bool Equals(object obj)
```

Comment:
I had to override Equals as well as GetHashCode method in order to be able to use Dictionary.